dev/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@khanacademy/perseus-dev-ui",
+    "description": "Perseus dev UI",
+    "author": "Khan Academy",
+    "license": "MIT",
+    "version": "0.0.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Khan/perseus.git",
+        "directory": "dev"
+    },
+    "scripts": {
+        "dev": "vite"
```

Comment:
I wonder if it'd be useful to put a `dev` script in the root `package.json` also so that we can `yarn dev` at the root.  it could just delegate to this `dev` script, but it'd remove the need to `cd dev` before `yarn dev`.